vignettes/EpiNow2.Rmd

```diff
@@ -293,3 +296,137 @@ estimates$summary$summary_plot
 ```
 
 ![plot of chunk plot_regional_epinow_summary](EpiNow2-plot_regional_epinow_summary-1.png)
+
+#### When regions have different settings
+
+Region-specific settings can be applied with the [`opts_list()`](https://epiforecasts.io/EpiNow2/reference/opts_list.html) function.
+
+As an example, let's use `reported_cases` from the previous example and define a 7-day random walk for testland and maintain the default for realland.
+
+
+``` r
+rt <- opts_list(
+  rt_opts(),
+  reported_cases,
+  testland = rt_opts(rw = 7) # Modify the default setting for testland
+)
+```
+
+Run the model
+
+``` r
+estimates <- regional_epinow(
+  data = reported_cases,
+  generation_time = gt_opts(example_generation_time),
+  delays = delay_opts(example_incubation_period + reporting_delay),
+  rt = rt,
+  gp = NULL,
+  stan = stan_opts(cores = 4, warmup = 250, samples = 1000)
+)
+#> INFO [2025-03-10 14:52:24] Producing following optional outputs: regions, summary, samples, plots, latest
+#> INFO [2025-03-10 14:52:24] Reporting estimates using data up to: 2020-04-21
+#> INFO [2025-03-10 14:52:24] No target directory specified so returning output
+#> INFO [2025-03-10 14:52:24] Producing estimates for: testland, realland
+#> INFO [2025-03-10 14:52:24] Regions excluded: none
+#> INFO [2025-03-10 14:52:34] Completed estimates for: testland
+#> INFO [2025-03-10 14:52:36] Completed estimates for: realland
+#> INFO [2025-03-10 14:52:36] Completed regional estimates
+#> INFO [2025-03-10 14:52:36] Regions with estimates: 2
+#> INFO [2025-03-10 14:52:36] Regions with runtime errors: 0
+#> INFO [2025-03-10 14:52:36] Producing summary
+#> INFO [2025-03-10 14:52:36] No summary directory specified so returning summary output
+#> INFO [2025-03-10 14:52:37] No target directory specified so returning timings
+```
+The summary looks like this
+
+``` r
+knitr::kable(estimates$summary$summarised_results$table)
+```
+
+
+
+|Region   |New infections per day |Expected change in reports |Effective reproduction no. |Rate of growth         |Doubling/halving time (days) |
+|:--------|:----------------------|:--------------------------|:--------------------------|:----------------------|:----------------------------|
+|realland |19142 (10733 -- 33387) |Increasing                 |1.2 (1.2 -- 1.3)           |0.054 (0.038 -- 0.068) |13 (10 -- 18)                |
+|testland |2019 (1034 -- 3973)    |Likely decreasing          |0.85 (0.61 -- 1.2)         |-0.04 (-0.11 -- 0.038) |-17 (18 -- -6.4)             |
+
+
+
+And the plots look like this.
+
+``` r
+estimates$summary$summary_plot
+```
+
+![plot of chunk plot_reg_epinow_opts_list_eg](EpiNow2-plot_reg_epinow_opts_list_eg-1.png)
+
+Another way to do this is by first defining the defaults for all regions with `opts_list()` and modifying individual components
```

Comment:
Do we want to give people this second workflow?

Reply:
This has been reverted in f11f846e3808726bb9b92e97a78449f85f7294af.